### PR TITLE
Added sidebar for molecule

### PR DESCRIPTION
**Pull request by pallavikadam-srijan**
Created sidebar for a molecule. 
Made change on ImageCard.stories.mdx file for update the spelling - **Image reveal card** instead of **Image revel card**

### CODE REVIEW

File: stories/Organism/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  useEffect(() => {
+    FaqFun();
+    SidebarMenu();
+  }, [])
+
+  return (
+    <div className='grid-x grid-margin-x'>
+      <div className='cell medium-4'>
+        <ul className='accordion-navbar' id="accordion">
+          <span className='accordion-menu hide-for-large'>{label}</span>
```
Comment (surbhi90) [inline stories/Organism/Sidebar/Sidebar.jsx:15]: @pallavikadam-srijan We can't have the span directly in ul and it should be wrapped in li

File: stories/Organism/Sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          <li className='accordion__item hide-for-small'>
```
Comment (surbhi90) [inline stories/Organism/Sidebar/Sidebar.jsx:16]: @pallavikadam-srijan It should be accordion-navbar__item

File: stories/Organism/Sidebar/sidebar.scss [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+    .accordion__item,
+    .hide-for-small {
+      display: block;
+    }
+  }
+}
+
+ul.accordion-navbar > li:nth-child(2),
+ul.accordion-navbar > li.accordion--active:nth-child(2) {
```
Comment (surbhi90) [inline stories/Organism/Sidebar/sidebar.scss:113]: @pallavikadam-srijan Don't use tags with classname or id

File: stories/Organism/Sidebar/Sidebar.jsx (hunk 1 above)
Comment (pallavikadam-srijan) [reply stories/Organism/Sidebar/Sidebar.jsx]: @surbhi90 Done

File: stories/assets/js/Faq_custom.js [hunk 4]
```diff
@@ -7,3 +7,9 @@ export function FaqFun() {
   });
 }
 /* Faq JS end custom */
+
+export function SidebarMenu () {
+  $(".hide-for-large").click(function() {
+    $(".accordion-navbar").toggleClass("show");
+  });
```
Comment (surbhi90) [inline stories/assets/js/Faq_custom.js:14]: @pallavikadam-srijan Please use the relevant id in JS

File: stories/Organism/Sidebar/Sidebar.jsx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+            <a href="#" className="accordion__heading">{headerText}</a>
+          </li>
+          <li className='accordion__no-children hide-for-small'>
+            <a href="#" className="accordion__heading">{headerText}</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  );
+};
```
Comment (surbhi90) [inline stories/Organism/Sidebar/Sidebar.jsx:49]: @pallavikadam-srijan Correct the structure

**Review by surbhi90 — CHANGES_REQUESTED** [review]
Please check the comments

File: stories/Organism/Sidebar/Sidebar.jsx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+export const Sidebar = ({ headerText, data, label }) => {
+  useEffect(() => {
+    FaqFun();
+    SidebarMenu();
+  }, [])
+
+  return (
+    <div className='grid-x grid-margin-x'>
+      <div className='cell medium-4 accordion-wrapper'>
+        <li className='accordion-menu' id='sidebar-menu'><span>{label}</span></li>
```
Comment (surbhi90) [inline stories/Organism/Sidebar/Sidebar.jsx:14]: @pallavikadam-srijan can you use li outside of ul and it's class should be accordion__menu as per BEM

File: stories/Organism/Sidebar/sidebar.scss [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+  font-size: $font-size-16;
+  padding: $spacing-16 0 $spacing-24 $spacing-24;
+  text-transform: uppercase;
+
+  @include devicebreak(medium) {
+    display: none;
+  }
+}
+
+.accordion-wrapper {
```
Comment (surbhi90) [inline stories/Organism/Sidebar/sidebar.scss:39]: @pallavikadam-srijan you have also define accordion-wrapper on the top also so can you please put all styling in one place only and please follow the standard like accordion and then using child like &__menu

File: stories/Organism/Sidebar/Sidebar.jsx [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+              })}
+            </div>
+          </li>
+          <li className='accordion__no-children'>
+            <a href="#" className="accordion__heading">{headerText}</a>
+          </li>
+          <li className='accordion-navbar__item'>
+            <a href="#" className="accordion__heading">{headerText}</a>
+            <div class="accordion__panel">
+              {data.map(function (item, index) {
```
Comment (surbhi90) [inline stories/Organism/Sidebar/Sidebar.jsx:32]: @pallavikadam-srijan Where we are using sidebar molecule

**Pull request merged**